schemas/dab.draft.schema.json

```diff
@@ -761,21 +759,15 @@
                             "oneOf": [
                               {
                                 "type": "string",
-                                "enum": [
-                                  "execute",
-                                  "*"
-                                ]
+                                "enum": ["execute", "*"]
                               },
                               {
                                 "type": "object",
                                 "additionalProperties": false,
                                 "properties": {
                                   "action": {
                                     "type": "string",
-                                    "enum": [
-                                      "execute",
-                                      "*"
-                                    ]
+                                    "enum": ["execute", "*"]
```

Comment:
any reason why all the enum values were moved to a single line as opposed to what they were previously - spread on multiple lines?